openshift/init_tenant.go

```diff
@@ -145,6 +144,7 @@ func do(config Config, callback Callback, username, usertoken string, templateVa
 // or default to the OOTB template included
 func loadTemplate(config Config, name string) ([]byte, error) {
 	teamVersion := config.TeamVersion
+	logCallback := config.LogCallback
```

Comment:
the only thing the controller `Callback` does is update the namespace in the database and do logging with almighty-core/log; which we don't need for the CLI tool (and the latter we can't build with ;)